apps/script/HoC2023AiGenerateWeights.py

```diff
@@ -0,0 +1,151 @@
+import spacy
```

Comment:
mind including a small comment indicating what this script is meant to do overall?